runtime/compiler/optimizer/J9Inliner.hpp

```diff
@@ -239,12 +239,16 @@ class TR_J9InlinerPolicy : public OMR_InlinerPolicy
 class TR_J9JSR292InlinerPolicy : public TR_J9InlinerPolicy
    {
    friend class TR_J9InlinerUtil;
+   friend class TR_J9InlinerPolicy;
```

Comment:
Why the friend class? couldn't the helper methods be moved up if they are needed in the regular and 292 policies?

Reply:
Thought conceptually JSR292 related queries should be in `TR_J9JSR292InlinerPolicy` but you are right this doesn't make sense in software design's point of view. Moved up the helper methods to TR_J9InlinerPolicy.